pretix_eth/models.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import annotations
```

Comment:
@kvbik This import was not about support for annotations.  It's about [this](https://www.python.org/dev/peps/pep-0563/), which is only available in python 3.7.  That's why I could do `WalletAddress` in stead of `'WalletAddress'` on line 35 below even though `WalletAddress` is declared after that line.  It doesn't throw a name error because evaluation of the annotation is deferred.

Reply:
Otherwise, I'm not sure why we should remove the annotation.  It's just more information to anyone who comes later to read the code.

Reply:
I know! Sorry for removing it without consulting with you @davesque ! Postponed Evaluation of Annotations is a great feature. Especially for Django where lot of declarations are kind of class-based. Unfortunately this is, as you say, available only in py3.7. Pretix itself on the other hand officially support py3.6 and I was thinking it would be nice for a plugin to reflect that. Also py3.6 is the official python3 of LTS edition of Ubuntu 18.04. As I am great fan of all the great things py3.7+, running this in production with less hassle was the main motivation. If you agree to keep support for py3.6 - I am not against leaving some more verbose comments on this place and of course adding it back while dropping support of py3.6.

Reply:
I was trying to find a way of re-organising the code little bit to have the annotation possible in py3.6, but no luck.

Reply:
Oh, do you mean the deferred evaluation future import is not available in 3.6? I guess that could be. I can't remember. If that's the case, you can still keep the annotation. It just needs to be `'WalletAddress'` instead of `WalletAddress`.

Reply:
I am testing the annotation with quotes right now, if it works I will send a PR asap.

Reply:
let's move to #116 for further discussion